cvat-ui/src/index.html

```diff
@@ -5,17 +5,21 @@
 -->
 
 <!DOCTYPE html>
-    <html lang="en">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="X-UA-Compatible" content="ie=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <meta name="description" content="Computer Vision Annotation Tool (CVAT) is a free, open source, web-based image and video annotation tool which is used for labeling data for computer vision algorithms. CVAT supports the primary tasks of supervised machine learning: object detection, image classification, and image segmentation. CVAT allows users to annotate data for each of these cases" />
-        <meta name=”robots” content="index, follow">
-        <title>Computer Vision Annotation Tool</title>
-    </head>
+<html lang="en">
 
-    <body>
-        <div id="root"></div>
-    </body>
-</html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description"
+        content="Computer Vision Annotation Tool (CVAT) is a free, open source, web-based image and video annotation tool which is used for labeling data for computer vision algorithms. CVAT supports the primary tasks of supervised machine learning: object detection, image classification, and image segmentation. CVAT allows users to annotate data for each of these cases" />
+    <meta name=”robots” content="index, follow">
+    <title>Computer Vision Annotation Tool</title>
+</head>
+
+<body>
+    <div id="root"></div>
+    <div id="layout-grid"></div>
```

Comment:
In CVAT code somewhere we use z-index. Could it be an issue?

Reply:
Nope, because the grid must be on top of all other. It has a higher z-index